pinot-common/src/main/java/org/apache/pinot/common/request/context/ExpressionContext.java

```diff
@@ -74,8 +83,27 @@ public LiteralContext getLiteral(){
     return _literal;
   }
 
-  public String getIdentifier() {
-    return _identifier;
+  // Please check that the _type is Identifier before calling these functions.
+  public String getIdentifierName() {
+    if (_identifier == null) {
+      return null;
+    }
+
+    return _identifier.getName();
+  }
+
+  public int getIdentifierIndex() {
```

Comment:
Can we make this change contained in v2 (in NewAggregateOperator) so that we don't need this interface change?